Update tutorial.md (#621)
open-spaced-repetition/fsrs4anki: docs/tutorial.md

### Learning and re-learning steps

> When FSRS is enabled, the learning and re-learning steps should be chosen in such a way that all the learning steps can be completed on the same day. This means that steps longer than or **equal** to 1 day should not be used. `23h` is not recommended because, while it's technically less than one day, it's very unlikely that you will be able to finish this step on the same day as your first review. In general, any steps longer than 12-14 hours are not recommended, since most people will not be able to finish such steps within the same day. Examples of good learning steps: 10m, 15m, 20m, 30m.

Reviewer: ```suggestion
When FSRS is enabled, the learning and re-learning steps should be chosen in such a way that all the learning steps can be completed on the same day. This means that steps longer than or **equal** to 1 day should not be used. `23h` is not recommended because, while it's technically less than one day, it's very unlikely that you will be able to finish this step on the same day as your first review. In general, any steps longer than 12-14 hours are not recommended, since most people will not be able to finish such steps within the same day. A reasonable learning step can be 10m, 15m, 20m or 30m.
```

> When FSRS is enabled, the learning and re-learning steps should be chosen in such a way that all the learning steps can be completed on the same day. This means that steps longer than or **equal** to 1 day should not be used. `23h` is not recommended because, while it's technically less than one day, it's very unlikely that you will be able to finish this step on the same day as your first review. In general, any steps longer than 12-14 hours are not recommended, since most people will not be able to finish such steps within the same day. A reasonable learning step can be 10m, 15m, 20m or 30m.

Reviewer: ```suggestion
When FSRS is enabled, the learning and re-learning steps should be chosen in such a way that all the learning steps can be completed on the same day. In general, any steps longer than 12-14 hours are not recommended because most people will not be able to finish such steps within the same day. A reasonable learning step can be 10m, 15m, 20m or 30m.
```

Author: I added that part because some user thought that 23h is a good learning step

Reviewer: This section is quite repetitive. The suggested wording retains the essence while significantly decreasing the length of the text.

Author: We need to clarify what "can be completed on the same day" means

Reviewer: I have updated the comment. Have a look again.

The reason is that FSRS can determine more optimal intervals but the use of longer (re)learning steps doesn't allow FSRS to schedule the reviews, making the scheduling less optimal. In addition, if longer steps are used, there can be cases where the "Hard" interval exceeds the "Good" interval. 

> Secondly, the use of multiple short (re)learning steps is also discouraged. This is because research shows that same-day reviews have a negligible impact on long-term memory. However, if you notice that your retention for young cards is much lower than desired, it may be advisable to add several learning steps, for example, 2h or 4h.

Reviewer: ```suggestion
Secondly, the use of multiple short (re)learning steps is also discouraged. This is because research shows that same-day reviews have a negligible impact on long-term memory. However, if you notice that your retention for young cards is much lower than desired, adding more intraday learning steps (such as 2h or 4h) may be helpful.
```

### Learning and re-learning steps

> When FSRS is enabled, the learning and re-learning steps should be chosen in such a way that all the learning steps can be completed on the same day. This means that steps longer than or **equal** to 1 day should not be used. `23h` is not recommended because, while it's technically less than one day, it's very unlikely that you will be able to finish this step on the same day as your first review. In general, any steps longer than 12-14 hours are not recommended, since most people will not be able to finish such steps within the same day. Examples of good learning steps: 10m, 15m, 20m, 30m.

Reviewer 2: The word `good` seems too strong here. How about `acceptable`?
